en/faq/nginx-proxy.md

```diff
@@ -35,6 +35,80 @@ server {
 }
 ```
 
+# Using nginx with generated pages and a caching proxy as fallback
+
+If you have a high volume website with regularly changing content, you might want to benefit from Nuxt generate capabilities and [nginx caching](https://www.nginx.com/blog/nginx-caching-guide).
+
+Below is an exampe configuration which you will most likely need to adapt to your needs.
+
+```nginx
+proxy_cache_path  /PATH/TO/YOUR/CACHE levels=1:2 keys_zone=nuxt-cache:25m max_size=1g inactive=60m use_temp_path=off;
```

Comment:
Trust me. Most people love to copy paste it :D Add a default real path.

Reply:
Ok, will use the same one as the nginx docs